services/api/src/resources/environment/helpers.ts

```diff
@@ -27,6 +27,19 @@ export const Helpers = (sqlClientPool: Pool) => {
   return {
     aliasOpenshiftToK8s,
     getEnvironmentById,
+    deleteEnvironment: async (name: string, pid: number) => {
+      // clean up environment variables
+      await query(
+        sqlClientPool,
+        'DELETE FROM `env_vars` WHERE `environment` = :pid',
```

Comment:
I don't think `environment = pid` is correct? You'll need to do an extra sql query to load the environment id, or use a `join` to the `environment` table, or change the function parameter to accept environment id instead of name, since in the resolver there is already a call to [load the full environment](https://github.com/uselagoon/lagoon/blob/main/services/api/src/resources/environment/resolvers.ts#L460)

Reply:
Ahh you are probably right, I'll have another look when I get a chance.